src/components/MessageInput/hooks/messageInput.js

```diff
@@ -687,8 +687,11 @@ export default function useMessageInputState(props) {
 
   const maxFilesLeft = maxFilesAllowed - numberOfUploads;
 
+  const isUploadEnabled = channel?.getConfig?.()?.uploads !== false;
```

Comment:
A small suggestion, as I think this line is a little bit tricky to read. Feel free to ignore it.

```suggestion
  const uploadConfig = channel?.getConfig?.()?.uploads;
  const isUploadEnabled = typeof uploadConfig === 'undefined' || uploadConfig === true;
```

Reply:
or it could be null as well 😅  maybe just adding a comment would do, so we're sure the undefined !== false was intended. x)

Reply:
agree, this is easier to read, but there are few more instances of `channel?.getConfig?.().X !== false` for other properties across the codebase, so let's keep this one similar to them. we can simplify them all together later